esmvalcore/preprocessor/_regrid.py

```diff
@@ -502,8 +502,71 @@ def regrid(cube, target_grid, scheme, lat_offset=True, lon_offset=True):
     See Also
     --------
     extract_levels : Perform vertical regridding.
+
+    Notes
+    -----
+    This preprocessor allows for the use of arbitrary Iris regridding schemes,
```

Comment:
```suggestion
    This preprocessor allows for the use of arbitrary :mod:`iris` regridding schemes,
```

Reply:
Hm. It's not realy about the `iris` module, is it? How about
```suggestion
    This preprocessor allows for the use of arbitrary :doc:`iris:index` regridding schemes,
```

Reply:
I'm not sure how this is rendered, so please just make the change. You can also ignore this comment if you like :smile:

Reply:
I made a change in the spirit and [here](https://esmvaltool--1448.org.readthedocs.build/projects/ESMValCore/en/1448/api/esmvalcore.preprocessor.html#esmvalcore.preprocessor.regrid) is how it renders. Let me know if you would like a further change.